Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 
 [dependencies]
 bevy = { version = "0.8.1", features = ["dynamic"] }
+rand = "0.8.4"
```

Comment:
Is Farnan okay with us using the rand crate?

Reply:
Yes.